### DISCUSSION TITLE
Updating getFillColor with updateTriggers

### DISCUSSION THREAD

**Discussion by Kris-B** [visgl/deck.gl#5701] [Q&A] [answered]
I'm trying to change the fillcolor of elements of one geojson layer dynamically.
I'm using ``updateTriggers``, but I can't find the right way to implement it.

Maybe someone could have a look at this sample : https://codepen.io/Kris-B/pen/YzNgLWo?editors=1011

Any help would be appreciated !

**Comment by sclebal** [comment, marked as answer]
Hi! You have to clone the layer updating the updateTriggers property `layer = layer.clone({ updateTriggers: { getFillColor: showAll }})` before setProps.

**Reply by Kris-B** [reply]
Great, many thanks for the quick answer!
